.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,31 @@
+{
+	"name": "devcontainer-fluent-bit",
+	"image": "mcr.microsoft.com/devcontainers/base:jammy",
+
+	// Features to add to the dev container. More info: https://containers.dev/features.
+	"features": {
+		"ghcr.io/devcontainers/features/docker-in-docker:2": {
```

Comment:
Why do we need this?

Reply:
it allows to use docker in devcontainer. as a result, if a developer want to build a docker image, he/him can build with dind in this environment. we can customize it but the environement should allow to use docker / container runtime. maybe we can replace it with daemonless build tool like kaniko in future.

Reply:
Sure, my question was more let's add a comment in case anyone asks. :-)